tests/data/mock-github-api-v3-response.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "mock-repository",
+  "issues_url": "http://localhost:8080/tests/data/mock-github-api-v3-issue-response.json",
```

Comment:
If this URL is going to be used, it might be a problem. There is no assurance that "ttp://localhost:8080" will be available on Travis or elsewhere.

Reply:
Oh, that's the reason why Travis is failing. But I couldn't find a way to change it to something like window.location as this is a JSON file

Reply:
yeah :) I've change how issues-and-notes.js works now, so it doesn't download the index from github anymore. So now it should work... but you will need to rebase this.... sorry about the merge conflict 😭

Reply:
by "it should work", I mean, it should be easier to test.

Reply:
So all in all what should I do, only a rebase?

Reply:
@marcoscaceres, I like the conflict :laughing:

Reply:
@himanish-star let's get this back to a good state, just showing your changes. Then we can work through it.

Reply:
Actually @marcoscaceres. I'm outside my home and will get back to work in the night after 2hrs

Reply:
@marcoscaceres, stuck now for a while
![image](https://user-images.githubusercontent.com/30361728/37248579-a4f67f94-24fb-11e8-8de0-0caddb24ac72.png)

Reply:
Also @marcoscaceres, I'll manage to update the code as you want, as now I can clearly see the new changes and how to use them. In the meanwhile it is my sincere request to you, to please find some time from your busy schedule to review my proposal today :pray:

Reply:
@marcoscaceres, I have pushed the changes and there's only one problem now, that is resolving to route `$(window.location.origin)/tests/data/issues/1540` won't fetch the JSON file we'll have to resolve to `$(window.location.origin)/tests/data/issues/1540.json`